models/cart.model.js

```diff
@@ -0,0 +1,11 @@
+const { Schema, model } = require("mongoose");
+
+const CartSchema = new Schema({
+  products: [
```

Comment:
This is a good way to define the products model. This way its easier for updating the quantity. I can also try to implement it in this way so that some issues of my cart gets resolved

Reply:
Great. You can ping me if you need any help.